content/blog/2018/06/2018-06-15-anchore-image-scanning.adoc

```diff
@@ -0,0 +1,173 @@
+---
+:layout: post
+:title: "Securing your Jenkins CI/CD Container Pipeline with Anchore (in under 10 minutes)"
+:tags:
+- events
+- community
+- developer
+:author: michaelneale
+---
+
+As more and more Jenkins users ship docker containers, it is worth thinking about the security implications of this model, where the variance in software being included by developers has increased dramatically from previous models. Security implications in this context include what makes up the image, but also the components of the app that get bundled into your image. Docker images are increasingly becoming a “unit of deployment”, and if you look at a typical app (especially if it is a microservice), much of the components, libraries, and system are someone else’s code. 
+
+
+link:https://github.com/anchore/anchore-engine[Anchore] exists to provide technology to act as a last line of defense, verifying the contents of these new deployable units against user specified policies to enforce security and compliance requirements. In this blog you will get a quick tour of this capability, and how to add the open-source Anchore Engine API service into your pipeline to validate that the flow of images you are shipping comply with your specific requirements, from a security point of view. 
+
+image:/images/post-images/2018-06-15-anchore/anchore-pipeline.png[title="Sample Failure", role="center", width=350]
+
+Key among the fundamental tenets of agile development is the notion of “fail fast, fail often”, which is where CI/CD comes in: A developer commits code into the source code repository, such as github, that automatically triggers Jenkins to perform a build of the application that is then run through automated tests. If these test fails the developer is notified immediately and can quickly correct the code. This level of automation increases the overall quality of code and speeds development.
+
+
+While some may feel that “fail fast” sounds rather negative (especially regarding security), you could better describe this process as “learn fast” as mistakes are found earlier in the development cycle and can be easily corrected.
+
+
+The increased use of CI/CD platforms such as Jenkins has helped to improve the efficiency of development teams and streamlined the testing process and we can leverage the same CI/CD infrastructure to improve the security of our container deployments.
+
+For many organizations the last step before deploying an application is for the security team to perform an audit, this may entail scanning the image for vulnerable software components, for example outdated operating system packages that contain known security vulnerabilities (CVEs), to verify that the applications and operating systems are correctly configured and that the organization’s best practices and compliance policies have been correctly implemented.
```

Comment:
Also a run-on sentence. My attempt at a rewrite:
'For many organizations the last step before deploying an application is for the security team to perform an audit. This may entail scanning the image for vulnerable software components (like outdated packages that contain known security vulnerabilities) and verifying that the applications and OS are correctly configured. They may also check that the organization’s best practices and compliance policies have been correctly implemented.'